tests/distributions/test_transforms.py

```diff
@@ -136,7 +136,7 @@ def test_poly_jacobians(self):
 
     def test_householder_inverses(self):
         for input_dim in [2, 5, 10]:
-            self._test_inverse(input_dim, T.householder(input_dim, count_transforms=2))
+            self._test_inverse([input_dim], T.householder(input_dim, count_transforms=2))
```

Comment:
This probably needs to be a tuple or `torch.Size` e.g. `(input_dim,)`.

Reply:
I've realized it works if I leave `input_dim` as is in all the other test methods, and use `shape` for `GeneralizedChannelPermute`